ansible/roles/spdashboard/tasks/install-branch.yml

```diff
@@ -40,7 +40,7 @@
 - name: Delete config
   file: path={{spdashboard_data_dir}}/current/app/{{ item }} state=absent
```

Comment:
Please discuss this with @quartje, but I think we can let go of the `/ansible` roles in the project. At the very least, our dev roles can be removed as we use the Docker solution exclusively nowadays.

Reply:
Confirmed by @quartje remove ansible/vagrant